<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QMainWindow" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>691</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="dept_major">
     <attribute name="title">
      <string>院系专业</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>110</y>
        <width>91</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QPushButton" name="add_dept">
         <property name="text">
          <string>添加院系</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="del_dept">
         <property name="text">
          <string>删除院系</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>280</y>
        <width>91</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QPushButton" name="add_major">
         <property name="text">
          <string>添加专业</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="del_major">
         <property name="text">
          <string>删除专业</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>211</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>院系</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="depts"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>240</y>
        <width>211</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>专业</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="majors"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="teacher">
     <attribute name="title">
      <string>教师管理</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_5">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>50</y>
        <width>221</width>
        <height>28</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="teacher_search_edit"/>
       </item>
       <item>
        <widget class="QPushButton" name="teacher_search_btn">
         <property name="text">
          <string>搜索教师号/教师名</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>90</y>
        <width>54</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>新增教师</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_6">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>261</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>院系</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="teacher_depts_combo"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>教师</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="teacher_show_list"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>54</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>浏览教师</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_7">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>120</y>
        <width>256</width>
        <height>221</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>院系</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="teacher_tdept_combo"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>教师名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="teacher_tname_edit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>教师号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="teacher_tno_edit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>教师密码</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="teacher_tpwd_edit">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QPushButton" name="teacher_delete_btn">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="teacher_modify_btn">
           <property name="text">
            <string>修改</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="teacher_add_btn">
           <property name="text">
            <string>添加</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <zorder>layoutWidget_5</zorder>
     <zorder>layoutWidget_6</zorder>
     <zorder>layoutWidget_7</zorder>
     <zorder>label_11</zorder>
     <zorder>label_13</zorder>
    </widget>
    <widget class="QWidget" name="student">
     <attribute name="title">
      <string>学生管理</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>60</y>
        <width>54</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>新增学生</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>54</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>浏览学生</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_8">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>231</width>
        <height>261</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>院系</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="student_depts_show_combo"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_18">
         <item>
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>专业</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="student_majors_show_combo"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>学生</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="student_show_list"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_9">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>201</width>
        <height>28</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLineEdit" name="student_search_edit"/>
       </item>
       <item>
        <widget class="QPushButton" name="student_search_btn">
         <property name="text">
          <string>搜索学号/学生名</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_10">
      <property name="geometry">
       <rect>
        <x>371</x>
        <y>80</y>
        <width>281</width>
        <height>261</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>院系</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="student_sdept_show_combo"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_23">
         <item>
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string>专业</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="student_smajor_show_combo"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_22">
         <item>
          <widget class="QLabel" name="label_33">
           <property name="text">
            <string>密码</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="student_spwd_edit">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_20">
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>学号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="student_sno_edit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>年级</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="student_sgrade_combo"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_21">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>姓名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="student_sname_edit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QPushButton" name="student_delete_btn">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="student_modify_btn">
           <property name="text">
            <string>修改</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="student_add_btn">
           <property name="text">
            <string>添加</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <zorder>layoutWidget_8</zorder>
     <zorder>layoutWidget_9</zorder>
     <zorder>layoutWidget_10</zorder>
     <zorder>label_4</zorder>
     <zorder>label_5</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
